### DISCUSSION TITLE
Interface conditions: How to use specific set of points (anchors) in PDE function

### DISCUSSION THREAD

**Discussion by lucabeetz** [lululxvi/deepxde#748] [Q&A]
Hey, thank you for creating and maintaining this great library. 

I have a geometry with two different materials and interface conditions which I'd like to enforce at the boundaries of the two materials. I have defined the regions of the materials using geometry elements (Rectangles) and would like to use `random_boundary_points` to sample points at the interfaces. I would like to use this specific set of points in the PDE function to calculate the interface loss and am wondering how to do this. I know that one can add training points as `anchors` but I am unsure how one could then access these in the PDE function, which only takes `x` and `y` as arguments.

Please tell me if I should clarify anything and I'd greatly appreciate any suggestions on how to best do this.

**Comment by lululxvi** [comment]
Yes, use `anchors` to provide the interface points. Why do you need access these points in the PDE? The PDE loss is applied for all points.

**Reply by lucabeetz** [reply]
My current setup consists of two different materials with different PDE constants. The network predicts two values, one value for each material. Then I use a mask (based on the geometry) to select which output should be used for the loss of each point.

My idea to enforce interface conditions at the boundaries of these regions would require me to get the output of the network at points on the boundary. These predictions (their derivatives) should match and they thus create an additional loss.

**Reply by lululxvi** [reply]
Use `OperatorBC` for the interface condition.

**Comment by w-venton** [comment]
@lucabeetz  hi, have you solved your problem? I am also facing a question like yours, can you share your code? Thank you very very much!
